wp-content/themes/core/blocks/core/postauthorname/style.pcss

```diff
@@ -0,0 +1,25 @@
+/**
+ * Styles specific to this block
+ */
+
+.wp-block-post-author-name {
+
+	@mixin t-body-small;
+
+	/**
+	 * add ::before pseduo element to create "By" text
+	 */
+	&::before {
+		content: "By ";
+	}
+
+	> a {
```

Comment:
Lets target the class `wp-block-post-author-name__link` here vs the tag for lower specificity